src/globus_cli/login_manager/manager.py

```diff
@@ -181,21 +177,22 @@ def assert_logins(self, *resource_servers, assume_gcs=False, assume_flow=False):
             )
 
     @classmethod
-    def requires_login(cls, *resource_servers: str):
+    def requires_login(cls, *resource_servers: ServiceNameLiteral):
```

Comment:
```suggestion
    def requires_login(cls, *services: ServiceNameLiteral):
```

Reply:
I clicked the "apply" button without thinking too hard and now the build is borked!

I'll do the requisite cleanup, but 125% agree with this change (as long as I update the usage sites for the argument to match! 😂 ).